Stactize.DurableFunctionOrchestratorExample.Tests/DurableOrchestratorTests.cs

```diff
@@ -144,18 +145,18 @@ public async Task RunOrchestratorWithEventSuspend_Should_CallActivity()
 
             var knownResultModel = _fixture.Create<OrchestrationResultModel>();
 
-            var context = new Mock<IDurableOrchestrationContext>();
+            var context = new Mock<TaskOrchestrationContext>();
             context.Setup(x => x.GetInput<OrchestrationActionModel>())
                    .Returns(knownActionModel);
-            context.Setup(x => x.CallActivityAsync<OrchestrationResultModel>(Constants.DurableActivity.Suspend, knownActionModel))
+            context.Setup(x => x.CallActivityAsync<OrchestrationResultModel>(Constants.DurableActivity.Suspend, knownActionModel, null))
```

Comment:
I think using `It.IsAny<TaskOptions>()` as a parameter instead of null for these Setup and Verify calls is more robust in case any options are added in the future.